plugins/woocommerce-blocks/tests/e2e/tests/product-collection/register-product-collection-tester.block_theme.spec.ts

```diff
@@ -393,6 +400,12 @@ test.describe( 'Product Collection registration', () => {
 			editor.canvas,
 			'A Test Product'
 		);
+		await expect(
+			block
+				.getByLabel( BLOCK_LABELS.productImage )
+				.locator( 'visible=true' )
```

Comment:
Do we need to use it because the locator matches some invisible images? 🤔

Reply:
It may be fine without, I can remove it. This was a bit of copy paste from other places where we look for the product images.

Reply:
Sounds good. To be clear, it's OK to keep as long as it helps avoid the hidden duplicate collision.